vagrant-spk

```diff
@@ -395,6 +395,86 @@ mkdir -p /var/run
 /usr/sbin/nginx -g "daemon off;"
 """
 
+DIY_SETUP_SCRIPT = r"""#!/bin/bash
+# This script is run in the VM once when you first run `vagrant-spk up`.  It is
+# useful for installing system-global dependencies.  It is run exactly once
+# over the lifetime of the VM.
+#
+# This is the ideal place to do things like:
+#
+#    export DEBIAN_FRONTEND=noninteractive
+#    apt-get install nginx nodejs nodejs-legacy python2.7 mysql-server
+#
+# If the packages you're installing here need some configuration adjustments,
+# this is also a good place to do that:
+#
+#    sed --in-place='' \
+#            --expression 's/^user www-data/#user www-data/' \
+#            --expression 's#^pid /run/nginx.pid#pid /var/run/nginx.pid#' \
+#            /etc/nginx/nginx.conf
+
+# By default, this script does nothing.  You'll have to modify it as
+# appropriate for your application.
+exit 0
+"""
+
+DIY_BUILD_SCRIPT = r"""#!/bin/bash
+# This script is run in the VM each time you run `vagrant-spk dev`.  This is
```

Comment:
Proposed addition to top:

`Purpose: Prepare the app code to be run, for example compiling it`

I am personally waffling about if I would put a `pip install`-type step in here, versus in the `setup`, but that mostly doesn't matter given that the Python platform support actually exists, and this is for DIY people anyway.